.github/workflows/ci.yml

```diff
@@ -140,6 +140,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        linux-only:
+          - ${{ github.event_name == 'pull_request' && !contains(github.head_ref, 'release-please') }}
```

Comment:
Let's add the label check here?

Reply:
And let's document it in the section "For project maintainers" https://pptr.dev/contributing#rolling-new-chrome-version